nest_elephant_tvb/translation/mpi_translator.py

```diff
@@ -3,192 +3,237 @@
 import time
 import numpy as np
 from mpi4py import MPI
-from threading import Thread, Lock
 from nest_elephant_tvb.translation.science_nest_to_tvb import store_data,analyse_data
 
-lock_status=Lock() # locker for manage the transfer of data from thread
-
 def init(path, param, comm, comm_receiver, comm_sender, loggers):
     '''
-    Initialize the translation with MPI. NOTE: more information will be added with more changes. Currently the multithreaded version is still in use.
+    Initialize the translation with MPI.
+    NOTE: more information will be added with more changes. This is still the very first version!
+    
+    TODO: IMPORTANT!!! MPI communication on NEST and TVB side is somewhat hardcoded.
+        --> NEST sends to and receives from rank 0 on transformer side. This works here, since rank 0 is the receiving rank
+            --> https://github.com/sdiazpier/nest-simulator/blob/ba0c9d637d6dce7bcdd09b526b28d20f815085d4/nestkernel/recording_backend_mpi.cpp
```

Comment:
This link it's better for the file because it track the modification of the actual file : 
https://github.com/sdiazpier/nest-simulator/blob/nest-i/nestkernel/recording_backend_mpi.cpp
For the input part, you can find it here : https://github.com/sdiazpier/nest-simulator/blob/nest-i/nestkernel/stimulating_backend_mpi.cpp